perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -74,6 +77,25 @@ def Prepare(benchmark_spec):
     #    invalid. Once the benchmark results are returned from Run() an
     #    invalid (or is that rather a 'valid' flag should be added.
     exit(1)
+  if FLAGS.against_device:
+    logging.info('Fill scratch disk on %s', vm)
+    command = (
+        ('sudo %s --filename=%s --ioengine=libaio '
+         '--name=fill-device --blocksize=512k --iodepth=64 '
+         '--rw=write --fill_device=1 --direct=1') %
```

Comment:
will --fill_device actually work for raw devices like /dev/sdb? I thought we tried it and it didn't work? 

Alternatively, can we just specify --size=device_size here and let fio write exactly that many bytes to the device? This will also allow people who want to save time and fill just part of the device (and use only that part for read testing).